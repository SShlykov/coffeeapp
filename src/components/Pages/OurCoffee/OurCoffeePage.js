import React from "react";
import Title from "./Title";
import FilterPanel from "../../FilterPanel";
import Items from "../../Items";

const OurCoffeePage = ({data, filter, onFilterChange, onSearchChange }) => {
    return(
        <section className="shop">
            <div className="container">
                <Title/>
                <FilterPanel onSearchChange={onSearchChange} filter={filter} onFilterChange={onFilterChange}/>
                <Items data={data}/>
            </div>
        </section>
    )
};

export default OurCoffeePage;