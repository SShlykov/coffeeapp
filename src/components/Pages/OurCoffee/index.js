import OurCoffee from "./ourCoffee";
export default OurCoffee;