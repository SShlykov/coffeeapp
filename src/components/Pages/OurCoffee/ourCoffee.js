import React from "react";
import girl from "./img/coffee_girl.jpg";
import { BeansLogo} from "../../SmallComp";

const OurCoffee = () => {
        return (
            <>
                <div className="row">
                    <div className="col-lg-4 offset-2">
                        <img className="shop__girl" src={girl} alt="girl"/>
                    </div>
                    <div className="col-lg-4">
                        <div className="title">About our beans</div>
                        <BeansLogo dark/>
                        <div className="shop__text">
                            Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                            <br/><br/>
                            Afraid at highly months do things on at. Situation recommend objection do intention<br/>
                            so questions. <br/>
                            As greatly removed calling pleased improve an. Last ask him cold feel<br/>
                            met spot shy want. Children me laughing we prospect answered followed. At it went<br/>
                            is song that held help face.
                        </div>
                    </div>
                </div>
                <div className="line"> </div>
            </>
        )
};
export default OurCoffee;