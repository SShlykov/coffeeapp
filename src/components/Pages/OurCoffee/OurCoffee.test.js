import React from 'react';
import OurCoffee from "./OurCoffeePage";
import {shallow} from 'enzyme';

describe("Testing <OurCoffee />", () => {
    const page = shallow(<OurCoffee/>);
    describe("Testing snap & state", ()=>{
        it("OurCoffee has rendered correctly", () => {
            expect(page).toMatchSnapshot();
        })
    })
})