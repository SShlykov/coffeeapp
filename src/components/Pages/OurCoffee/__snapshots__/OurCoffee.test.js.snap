// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <OurCoffee /> Testing snap & state OurCoffee has rendered correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OurCoffeePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <Title />
        <FilterPanel />
        <Items />
      </div>,
      "className": "shop",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Title />,
          <FilterPanel />,
          <Items />,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "filter": undefined,
            "onFilterChange": undefined,
            "onSearchChange": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "data": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <Title />
          <FilterPanel />
          <Items />
        </div>,
        "className": "shop",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Title />,
            <FilterPanel />,
            <Items />,
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "filter": undefined,
              "onFilterChange": undefined,
              "onSearchChange": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "data": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
